Tests/FunctionalTests/Sources/FTRScrollViewTest.m

```diff
@@ -28,6 +28,11 @@ - (void)setUp {
   [self openTestViewNamed:@"Scroll Views"];
 }
 
+
```

Comment:
nit: remove excess new line

Reply:
Have a habit of keeping two newline between methods. Removed in next commit.